superset/cli/native_filters.py

```diff
@@ -368,17 +368,15 @@ def cleanup(
             json_metadata = json.loads(dashboard.json_metadata or "{}")
             position_json = json.loads(dashboard.position_json or "{}")
 
-            if "native_filter_migration" not in json_metadata:
-                click.echo(f"{str(dashboard)} has not been upgraded")
-                continue
-
             # Remove the saved filter configurations.
-            del json_metadata["native_filter_migration"]
-            dashboard.json_metadata = json.dumps(json_metadata)
+            if "native_filter_migration" in json_metadata:
+                del json_metadata["native_filter_migration"]
+                dashboard.json_metadata = json.dumps(json_metadata)
 
             for value in position_json.values():
                 if (
                     isinstance(value, dict)
+                    and value["type"] == "MARKDOWN"
```

Comment:
```suggestion
                    and "meta" in value
```

Reply:
@michael-s-molina I've updated the PR to use a variant of this (in the form of the walrus operator) throughout for consistency.